### DISCUSSION TITLE
Install Embedded Milvus -> milvus.start() err

### DISCUSSION THREAD

**Discussion by changjiale** [milvus-io/milvus#30783] [Q&A and General discussion]
milvus.start() 

>>> ---Milvus Proxy successfully initialized and ready to serve!---
panic: insertBufferNode flushBufferData failed, err = All attempts results:
attempt #1:can't add empty string into payload
attempt #2:can't add empty string into payload
attempt #3:can't add empty string into payload
attempt #4:can't add empty string into payload
attempt #5:can't add empty string into payload

goroutine 1064 [running]:
github.com/milvus-io/milvus/internal/datanode.(*insertBufferNode).Operate(0xc0021bc2d0, {0xc0026365d0, 0xc002043a00, 0x36})
        /Users/binbin/embeded_milvus/milvus/internal/datanode/flow_graph_insert_buffer_node.go:403 +0x3119
github.com/milvus-io/milvus/internal/util/flowgraph.(*nodeCtx).work(0xc00267fe00)
        /Users/binbin/embeded_milvus/milvus/internal/util/flowgraph/node.go:102 +0x1fd
created by github.com/milvus-io/milvus/internal/util/flowgraph.(*nodeCtx).Start
        /Users/binbin/embeded_milvus/milvus/internal/util/flowgraph/node.go:70 +0x87

**Comment by changjiale** [comment]
python3 -m pip install milvus==2.1.0

python version:  3.10.13

**Reply by yhmo** [reply]
The version v2.1.0 is too old, and is no longer maintained.
The recommended lightweight milvus is: https://github.com/milvus-io/milvus-lite. Milvus v2.3.5 is integrated in the latest milvus-lite. pymilvus v2.3.x is required.

```
python3 -m pip install milvus
python3 -m pip install pymilvus==2.3.6
```
